src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpWhosInControlApplicationController.cs

```diff
@@ -74,8 +84,59 @@ public async Task<IActionResult> StartPage(Guid applicationId)
             return await AddPeopleInControl(applicationId);
         }
 
+        [HttpGet("refresh-directors-pscs")]
+        public async Task<IActionResult> RefreshDirectorsPscs(Guid applicationId, string ukprn, string companyNumber)
+        {
+            try
+            {
+                var companyDetails = await _companiesHouseApiClient.GetCompanyDetails(companyNumber);
```

Comment:
Comment - Does this api call handle null ok?

Reply:
discussed that this pattern was lifted from elsewhere so better to keep with the existing behaviour for now...